docs/content/ja/docs/user-guide/secret-management.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "機密データの管理"
+linkTitle: "機密データの管理"
+weight: 6
+description: >
+  Git リポジトリ内の機密データの管理方法について
+---
+
+全てを Git で管理するのは便利ですが、Kubernetes の Secret や Terraform の Credential などといった機密データを Git に直接格納するのは安全ではありません。
+ここでは、そういった機密情報をどのように Git 上で安全に管理できるかについて説明します。
+
+基本的には以下のフローになっています。
+- PipeCD の Web コンソールで暗号化した機密情報を Git に格納
+- `Piped` は Deployment のタスクを実行する際に復号化
+
+## 前提条件
+
+この機能を使う為には RSA の公開鍵と秘密鍵のペアが必要になり、下記のコマンドで作ることができます。
```

Comment:
It doesn't have to be an RSA key pair, so let's remove it.
I opened a patch to fix the original: https://github.com/pipe-cd/pipe/pull/2891